flexmeasures/data/models/planning/tests/test_solver.py

```diff
@@ -353,14 +379,24 @@ def test_building_solver_day_2(
         assert soc >= max(soc_min, battery.get_attribute("min_soc_in_mwh"))
         assert soc <= battery.get_attribute("max_soc_in_mwh")
 
-    # Check whether the resulting soc schedule follows our expectations for 8 expensive, 8 cheap and 8 expensive hours
+    # Check whether the resulting soc schedule follows our expectations for:
```

Comment:
"... our expectations. To recap, per scenario 1 and 2, the schedule should mainly be influenced by:"